code/obj/item/implant.dm

```diff
@@ -1099,20 +1155,23 @@ THROWING DARTS
 
 /obj/item/implanter/mindslave
 	icon_state = "implanter1-g"
+	only_in = "head"
```

Comment:
Not a fan. Noob trap.

Reply:
This is something I wanted to keep, but I'm not going to hold up _this_ PR just to get it.